tutorials/3d/portals/intermediate.rst

```diff
@@ -0,0 +1,80 @@
+Intermediate
+============
+Normally, when you use Godot, all objects that you can see (:ref:`VisualInstance<class_VisualInstance>`\ s) are treated in the same way by the engine. The portal renderer is slightly different, in that it makes a distinction between the different roles objects will have in your game, in order to define the :ref:`Room<class_Room>`\ s, and in order to render and process everything in the most efficient way.
+
+Portal mode
+~~~~~~~~~~~
+If you look in the inspector, every VisualInstance in Godot is derived from a :ref:`CullInstance<class_CullInstance>`, where you can set a ``PortalMode``. This determines how objects will behave in the portal system.
+
+.. image:: img/cull_instance.png
+
+STATIC
+^^^^^^
+The default mode for objects is ``STATIC``. Static objects are objects within rooms that will not move throughout the life of the level. Things like floors, walls, ceilings are good candidates for ``STATIC`` objects.
+
+DYNAMIC
+^^^^^^^
+Dynamic mode is for objects that are expected to move during the game. But there is a limitation - they must not move outside of their original room. These objects are handled very efficiently by the system. Examples might include moving platforms, and elevators.
+
+ROAMING
+^^^^^^^
+Roaming mode is for objects that can move between rooms. Things like players you will want to be roaming. These are more expensive to calculate than ``STATIC`` or ``DYNAMIC`` modes, because the system has to keep track of which room a roaming object is within.
```

Comment:
```suggestion
ROAMING
^^^^^^^

Roaming mode is for objects that can move between rooms. Things like players and enemies should be marked as roaming. These are more expensive to calculate than ``STATIC`` or ``DYNAMIC`` modes, because the system has to keep track of which room a roaming object is within.
```